owncloudApp/build.gradle

```diff
@@ -135,6 +146,16 @@ android {
     }
 }
 
+def getDebugValueFromStrings(String value) {
+    try {
+        def stringsFile = file("./src/main/res/values/setup.xml")
+        return new XmlParser().parse(stringsFile).string.find { it.@name.equals value }.text()+".debug"
+    }catch(e){
+        println(e)
+        return "default.packagename"
```

Comment:
Can't we use the real package name here instead of hardcoding it?

Reply:
I `thow` it now